src/encoded/types/dataset.py

```diff
@@ -466,6 +466,48 @@ def files(self, request, original_files, status):
     def superseded_by(self, request, superseded_by):
         return paths_filtered_by_status(request, superseded_by)
 
+    @calculated_property(condition='contributing_files', schema={
+        "title": "Biochemical profile inputs",
+        "description": "The data used to generate a cCRE annotation for a given cell type or biosample.",
```

Comment:
since cCREs may be cell type agnostic, perhaps you can make less definitive description? Maybe something like: "The input data used to generate the cCRE annotation." ?

Reply:
Sounds good, I altered it (this is closer to what I have in the changelog also)